typescript/packages/agents-m365copilot-core/package.json

```diff
@@ -3,8 +3,8 @@
     "access": "public"
   },
   "name": "@microsoft/agents-m365copilot-core",
-  "version": "1.0.0-preview.1",
-  "description": "Core functionalities for the Microsoft Agents M365 Copilot JavaScript SDK",
+  "version": "1.0.0-preview.2",
```

Comment:
release please should be managing this

Reply:
Ack. Will apply the discussed changes in #84